test_addict.py

```diff
@@ -357,6 +357,12 @@ def test_update_with_lists(self):
         self.assertDictEqual(org, correct)
         self.assertIsInstance(org.b[0], dict)
 
+    def test_update_with_kws(self):
+        org = Dict(one=1, two=2)
+        someother = Dict(one=3)
+        someother.update(one=1, two=3)
```

Comment:
I changed the test to make it fail and I committed the failing version ;). Now fixed.